src/OrbitGl/App.cpp

```diff
@@ -1773,16 +1745,15 @@ void OrbitApp::LoadPreset(const PresetFile& preset_file) {
       SendWarningToUi("Preset only partially loaded",
                       absl::StrFormat("The following modules were not loaded:\n* %s",
                                       absl::StrJoin(module_paths_not_found, "\n* ")));
-      //    } else {
-      //      // Then if load was successful and the preset is in old format - convert it to new
-      //      one. auto convertion_result = ConvertPresetToNewFormatIfNecessary(preset_file); if
-      //      (convertion_result.has_error()) {
-      //        SendWarningToUi(
-      //            "Conversion to new format failed",
-      //            absl::StrFormat("Unable to convert preset file \"%s\" to new file format: %s",
-      //                            preset_file.file_path().string(),
-      //                            convertion_result.error().message()));
-      //      }
+    } else {
+      // Then if load was successful and the preset is in old format - convert it to new one.
+      auto convertion_result = ConvertPresetToNewFormatIfNecessary(preset_file);
+      if (convertion_result.has_error()) {
+        SendWarningToUi(
+            "Conversion to new format failed",
```

Comment:
Should we really show this to the user?

Reply:
Converted to ERROR, also added the code to restore backed up file in case SaveToFile fails.

Reply:
not sure where my comment went. I converted it to ERROR and added "restore from .backup" functionality to ConvertPresetToNewFormatIfNeccessary().